source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Create an Advanced Composer-managed WordPress Site 
+subtitle: Create an Advanced Composer-managed WordPress Site 
+description: Learn how to create a WordPress Site that uses Composer, Continuous Integration, and Automated Testing on Pantheon
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/create-wp-site-composer-ci-auto-test
+anchorid: create-wp-site-composer-ci-auto-test
+---
+
+This section provides steps to create a new Pantheon WordPress site that will use [Composer](https://getcomposer.org/), Continuous Integration, and Automated Testing. This guide will get you started, but you will need to customize and maintain the CI/testing set up for your projects.
+
+## Requirements
+
+- [Terminus Build Tools Plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin)
+
+    - You must use the [Build Tools 3.x release](https://github.com/pantheon-systems/terminus-build-tools-plugin/tree/3.x) if you are using **Terminus 3**.
+
+    - You must use the [Build Tools 2.x release](https://github.com/pantheon-systems/terminus-build-tools-plugin/tree/2.x) if you are using **Terminus 2**.
+
+- [PHP version](/php-versions#verify-current-php-versions) 7.2 or greater
+
+## Create Your Site
+
+1. Use the [Terminus Build Tools Plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin).
+
+    - The Terminus Build Tools plugin will scaffold the new project, including:
+
+       - A Git repository
+       - A free [Pantheon sandbox](/create-sites#sandbox-sites) site
+       - [Continuous Integration](https://pantheon.io/integrations/continuous-integration) configuration
+       - Credential set up
+
+1. Run the following command to spin up your site: 
+
+    ```bash
+    terminus build:project:create wp
+    ```
+
+    - This site will be based on the Pantheon-maintained [WordPress Composer repository](https://github.com/pantheon-systems/example-wordpress-composer).
+
+1. Review the sections below for important information about your site, including an explanation of the directory structure, Continuous Integration functions, and essential configuration actions.
+
+## Review Important Directories and Update File Paths
+
+### /web Directory
+
+Your site is stored and served from the `/web` subdirectory in the `pantheon.yml` file. You must store your website in this subdirectory for a Composer-based workflow. Placing your website in the subdirectory also allows you  to store tests, scripts, and other files related to your project in your repo without affecting your web document root. It also provides additional security by preventing web access through Pantheon. Your files may still be accessible from your version control project if it is public. See the [`pantheon.yml` documentation](/pantheon-yml#nested-docroot) for details.
```

Comment:
...from the `/web` subdirectory *next to* the `pantheon.yml` file.

It also provides additional security by preventing web access *to files outside of the document root* through Pantheon.